string Class
float64 probability
int64 xmin
int64 ymin
int64 width
int64 height

